Library/Homebrew/cmd/--cache.rb

```diff
@@ -10,10 +10,8 @@ module Homebrew
 
   extend Fetch
 
-  module_function
```

Comment:
`module_function` is incompatible with code that uses `extend`, because the methods are only available to the class methods. (We should probably remove these generally, just not in this PR…)